lib/_http_agent.js

```diff
@@ -305,6 +305,18 @@ Agent.prototype.removeSocket = function removeSocket(s, options) {
   }
 };
 
+Agent.prototype.keepSocketAlive = function keepSocketAlive(socket) {
+  socket.setKeepAlive(true, this.keepAliveMsecs);
+  socket.unref();
+
+  return true;
+};
+
+Agent.prototype.reuseSocket = function reuseSocket(socket, req) {
```

Comment:
Why is `req` passed in here? It appears to be unused. Is it for agents that override the function and want to use it?